### PR TITLE
LP-2967 Fix webinar schedule page

### PR DESCRIPTION
**Pull request by javeriaejaz-arbisoft**
[LP-2967](https://philanthropyu.atlassian.net/browse/LP-2967)

[Theme PR](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/###)

***

#### Description:
Add a description of your changes here.

***

## Things to Cross-check:

- [ ] Technical Docs updated if needed?
- [ ] [Domain Model](https://drive.google.com/file/d/1iRSqxDQQMXl6JzLw9nZtq1yGshwMBLTn/view?usp=sharing) updated if needed?
- [ ] [Core Changes Wiki](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes) updated if needed?

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3506/workflows/workflows_id/jobs/3506/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
-------------
Total:   4 lines
Missing: 1 line
Coverage: 75%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3507/workflows/workflows_id/jobs/3507/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3508/workflows/workflows_id/jobs/3508/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

File: openedx/adg/admin_dashboard/webinars/tests/test_forms.py [hunk 1]
```diff
@@ -23,15 +24,20 @@ def test_webinar_form_save(mocker, current_time, webinar_data):
...
-    assert webinar.audio_settings == Webinar.BOTH
+    if not is_valid_start_date:
+        webinar_data['start_time_year'] -= 1
+        assert not webinar_form.is_valid()
+    else:
+        assert webinar_form.is_valid()
+        webinar_form.save()
+
+        assert webinar.title == "Test Title"
+        assert webinar.description == "Test Description"
```
Comment (asadali145) [inline openedx/adg/admin_dashboard/webinars/tests/test_forms.py:36]: Use single quotes.

**Review by asadali145 — DISMISSED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/3509/workflows/workflows_id/jobs/3509/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/admin_dashboard/webinars/forms.py (75.0%): Missing lines 72
openedx/adg/admin_dashboard/webinars/tests/test_forms.py (100%)
-------------
Total:   17 lines
Missing: 1 line
Coverage: 94%
-------------
```

**Review by asadali145 — APPROVED** [review]

**Pull request merged**